PL2/PL2-gui/src/main/java/nl/tudelft/pl2016gr2/gui/view/graph/DrawComparedGraphs.java

```diff
@@ -225,26 +242,52 @@ private static void drawNode(Pane pane, HashMap<Integer, NodeCircle> circleMap,
   }
 
   /**
+   * Calculate the radius of the node. The readius depends on the amount of bases inside the node.
+   *
+   * @param node the node.
+   * @return the radius.
+   */
+  private static double calculateNodeRadius(GraphNodeOrder node) {
+    int amountOfBases = node.getNode().getSequenceLength();
+    double radius;
+    if (amountOfBases > 1000) {
+      radius = Math.log(amountOfBases) * 4.0 - 17.0;
```

Comment:
These are magic numbers. Also, some explanation about the choice of number would be nice.